pkg/crc/machine/start.go

```diff
@@ -108,30 +103,30 @@ func (client *client) Start(startConfig StartConfig) (StartResult, error) {
 					filepath.Base(startConfig.BundlePath),
 					bundleName))
 		}
-		vmState, err := host.Driver.GetState()
+		vmStatus, err := host.Driver.GetState()
```

Comment:
fwiw, after applying this PR, I still get
```
$ git grep vmState
pkg/crc/machine/machine.go:     vmState, err := host.Driver.GetState()
pkg/crc/machine/machine.go:             State:         vmState,
pkg/crc/machine/start.go:       vmState, err := host.Driver.GetState()
pkg/crc/machine/start.go:               Status:         vmState.String(),
```

Reply:
Dropped this commit